File: src/main/java/universalelectricity/core/CommonProxy.java
package universalelectricity.core;

import universalelectricity.core.net.NetworkTickHandler;
import net.minecraftforge.common.MinecraftForge;

public class CommonProxy
{
	public void init()
	{
		MinecraftForge.EVENT_BUS.register(NetworkTickHandler.INSTANCE);
	}
}

File: src/main/scala/universalelectricity/core/CommonProxy.scala
package universalelectricity.core

import cpw.mods.fml.common.FMLCommonHandler
import universalelectricity.core.grid.UpdateTicker

class CommonProxy
{
  def init
  {
    if (!UpdateTicker.useThreads)
      FMLCommonHandler.instance().bus().register(UpdateTicker)
  }
}